hooks/queries/votes/usePastVotes.ts

```diff
@@ -11,7 +11,7 @@ export function usePastVotes() {
     [pastVotesKey, roundId],
     () => getPastVotesAllVersions(),
     {
-      refetchInterval: (data) => (data ? false : 100),
+      refetchInterval: oneMinute,
```

Comment:
hitting the graph is cheap so might as well poll past votes (there is a delay between a round ending and the events being indexed by the graph)